include/deal.II/meshworker/dof_info.h

```diff
@@ -111,7 +111,8 @@ namespace MeshWorker
      * Constructor leaving the #block_info pointer empty, but setting the
      * #aux_local_indices.
      */
-    DoFInfo (const DoFHandler<dim, spacedim> &dof_handler);
+    DoFInfo(const DoFHandler<dim, spacedim> &dof_handler);
+    DoFInfo(const hp::DoFHandler<dim, spacedim> &dof_handler);
```

Comment:
Please add documentation for this constructor (which likely is similar to the constructor in the previous line).